test/util/expects_rejection.ts

```diff
@@ -0,0 +1,13 @@
+const expect = require('expect')
+
+export default function expectsRejection (res: Promise<any>): Promise<any> {
```

Comment:
@mslipper FYI https://github.com/domenic/chai-as-promised#assert-interface

Reply:
I don't think we're using `chai`... `expect` comes from Facebook's `jest` package and doesn't have up-to-date type bindings.

Reply:
Shall I add a task to convert over to `chai`? I think it'll be better if we do, given that we're not using Jest and the `expect` lib works best in tandem with `jest.

Reply:
That's a good idea. Let's agree on Mocha/Chai stack for testing.